capstone_project/src/test/java/com/google/sps/SchoolTest.java

```diff
@@ -14,31 +14,69 @@
 
 package com.google.sps;
 
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
 import com.google.sps.data.School;
+import com.google.sps.servlets.MapServlet;
+import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
 
-/** Class that tests the basic 'get' methods of School objects. */
+/** Class that tests the basic 'get' methods and equals operator of School objects. */
 @RunWith(JUnit4.class)
-public final class SchoolTest {
-
-  private final ArrayList<String> uciNews =
-      new ArrayList<String>(
-          Arrays.asList(
-              "https://news.uci.edu/2020/06/30/virtual-nurses-make-a-real-difference/",
-              "https://news.uci.edu/2020/07/07/uci-chancellor-emeritus-michael-v-drake-named-university-of-california-president/",
-              "https://news.uci.edu/2020/06/18/uci-podcast-jessica-millward-on-the-meaning-and-importance-of-juneteenth/"));
+public final class SchoolTest extends Mockito {
 
   private final double uciLatitude = 33.640339;
   private final double uciLongitude = -117.844248;
 
+  private DatastoreService datastore;
+  private MapServlet mapServlet;
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+
+  private LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    mapServlet = new MapServlet();
+    request = Mockito.mock(HttpServletRequest.class);
+    response = Mockito.mock(HttpServletResponse.class);
+
+    Entity schoolEntity = new Entity("School");
+    schoolEntity.setProperty("name", "UCI");
+    schoolEntity.setProperty("latitude", Double.toString(uciLatitude));
+    schoolEntity.setProperty("longitude", Double.toString(uciLongitude));
+    datastore.put(schoolEntity);
```

Comment:
Could you remove this section and where you test the UCI duplicate method, call doPost twice and check the count entity is still 1?